pages/en/lb2/Exposing-models-over-REST.md

```diff
@@ -322,7 +322,43 @@ MyModel.disableRemoteMethod('updateAttributes', isStatic);
 Be sure to call `disableRemoteMethod()` on your own custom model, not one of the built-in models; in the example below, for instance, the calls are `MyUser.disableRemoteMethod()` _not_ `User.disableRemoteMethod()`.
 " %}
 
-Here's an example of hiding all methods of the `MyUser` model, except for `login` and `logout`:
+
+
+Here's an example of hiding all methods of the `MyUser` model through configuration, except for `login` and `logout`:
+
+In `server/model-config.json`:
+
+```javascript
+"MyUser": {
+  "dataSource": "db",
+  "public": true,
+  "options": {
+    "remoting": {
+      "sharedMethods": {
+        "*": false,
+        "login": true,
+        "logout": true
+      }
+    }
+  }
+}
+```
+
+You can also configure all models through `config.json`'s `remoting` object:
```

Comment:
Can you please change this line to echo following message:
**You can also hide common methods across all models through `config.json`'s `remoting` object as follows:**